apps/backend/.env.example

```diff
@@ -46,3 +46,9 @@ MAILER_APP_PASSWORD=<GOOGLE_APP_PASSWORD>
 MAILER_HOST=smtp.gmail.com
 MAILER_PORT=465
 MAILER_SERVICE=Gmail
+
+#
+# OPEN AI
+#
+OPEN_AI_API_KEY=<OPEN_AI_API_KEY>
```

Comment:
Don't forget to notify others about these changes when this will be merged to main

Reply:
Sure